frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
Do we still need this line if we are using the rule `"react/no-unescaped-entities": "off",` ?

I didn't test it so I dont know 😕

If we don't need this line anymore could you remove it from all files you added it to?